ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-commands.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# The pullspec of an index image. Required.
+OO_INDEX="$OO_INDEX"
```

Comment:
minor nit: Since we have a place to document these in the step definition we might not need this stanza - it is a bit weird from a Bash perspective to see this line assigning a variable to itself. Might be easier as a comment:

```sh
# This script expects the following parameters as input:
# - OO_INDEX: the pullspec of an index image. Required.
# - OO_PACKAGE: ...
```

Reply:
It has a useful property of explicitly failing the `nounset` check when the step definition and actual script go out of sync for any reason. But I can add a comment about that.